tests/functional/mesh/reload_test.go

```diff
@@ -0,0 +1,266 @@
+package mesh
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	_ "github.com/fortytw2/leaktest"
+	"github.com/project-receptor/receptor/tests/functional/lib/mesh"
+	"github.com/project-receptor/receptor/tests/functional/lib/receptorcontrol"
+)
+
+func TestReload(t *testing.T) {
+	t.Parallel()
+	echoSleepLong := map[interface{}]interface{}{
+		"work-command": map[interface{}]interface{}{
+			"workType": "echosleeplong",
+			"command":  "bash",
+			"params":   "-c \"for i in {1..5}; do echo $i; sleep 3;done\"",
+		},
+	}
+	echoSleepLong50 := map[interface{}]interface{}{
+		"work-command": map[interface{}]interface{}{
+			"workType": "echosleeplong50",
+			"command":  "bash",
+			"params":   "-c \"for i in {1..5}; do echo $i; sleep 3;done\"",
+		},
+	}
+	echoSleepShort := map[interface{}]interface{}{
```

Comment:
your tests don't use echosleepshort or echosleeplong, so we can get rid of these definitions